### PR TITLE
Add ecl2csv and csv2ecl as ERT forward models

### PR DESCRIPTION
**Pull request by berland**
Add CSV2ECL and ECL2CSV as ERT  forward models.

No options are possible  through these, only DATAFILE and output file. Allowing options probably require dedicated forward models for each ecl2csv "subcommand".

### CODE REVIEW

File: ecl2df/hook_implementations/jobs.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+@hook_implementation
+@plugin_response(plugin_name="ecl2df")
+def installable_jobs():
+    return _get_jobs_from_directory("config_jobs")
+
+
+@hook_implementation
+@plugin_response(plugin_name="ecl2df")
+def installable_workflow_jobs():
+    return {}
```
Comment (sondreso) [inline ecl2df/hook_implementations/jobs.py:29]: This is not needed if it's not returning anything, but if this is needed in the future I'm fine with leaving it as is!

File: tests/test_ert_hooks.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+import pytest
+
+try:
+    from ert_shared.plugins.plugin_manager import ErtPluginManager
+except ImportError:
+    pytest.skip(
+        "ERT is not installed, skipping hook implementation tests.",
+        allow_module_level=True,
+    )
```
Comment (sondreso) [inline tests/test_ert_hooks.py:14]: I'm a little bit worried that if the module structure of ERT changes, this test will just be skipped and everything appears green. Maybe do something like this instead?
```python
try:
    import ert_shared
except ImportError:
    pytest.skip(
        "ERT is not installed, skipping hook implementation tests.",
        allow_module_level=True,
    )

from ert_shared.plugins.plugin_manager import ErtPluginManager
```

File: tests/test_hook_implementations.py [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+
+try:
+    import ecl2df.hook_implementations.jobs
+    from ert_shared.plugins.plugin_manager import ErtPluginManager
+except ImportError:
+    pytest.skip(
+        "ERT is not installed, or Python2. Skipping hook implementations.",
+        allow_module_level=True,
+    )
```
Comment (sondreso) [inline tests/test_hook_implementations.py:15]: Same as the other file :slightly_smiling_face:

**Review by sondreso — APPROVED** [review]
Other than my two minor comments, this LGTM!

File: ecl2df/hook_implementations/jobs.py (hunk 1 above)
Comment (berland) [reply ecl2df/hook_implementations/jobs.py:29]: Removed.

File: tests/test_ert_hooks.py (hunk 2 above)
Comment (berland) [reply tests/test_ert_hooks.py]: Fixed.

File: tests/test_hook_implementations.py (hunk 3 above)
Comment (berland) [reply tests/test_hook_implementations.py]: Fixed.

**Pull request merged**